### PR TITLE
Polish | Update to translation (Menu stuff & Gameplay)

### PR DESCRIPTION
**Pull request by MultiverseUncle**
# SCP:SL Translation PR Template

## Language
Polish

## Type of change
- [ ] Added new language translation
- [X] Changed a currently existing translation

## Have you tested the changes you've made?
- [ ] Yes
- [X] No (Unable to test it)

# Checklist:
- [X] I have made sure to not commit changes to other stuff than my own or the default english one
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [X] I have made sure that I'm not submitting a duplicate translation 
- [X] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

**Comment by MichalPetryka** [comment]
Those menu Translations feel really weird and unnatural for me, could you think a bit more about them?

**Comment by MultiverseUncle** [comment]
> Those menu Translations feel really weird and unnatural for me, could you think a bit more about them?

Do you mean the entire file or only these gameplay options?

**Comment by MichalPetryka** [comment]
Gameplay ones, for example "SCP-079 - PRZEŁĄCZ TRYB WIDOKU", something like "sposób zmiany widoku" would be better

**Comment by MultiverseUncle** [comment]
> Gameplay ones, for example "SCP-079 - PRZEŁĄCZ TRYB WIDOKU", something like "sposób zmiany widoku" would be better

Ahh, sorry. I though that it means something like "Change **deafult** view mode by pressing **button in options**"

File: Translations/pl/MenuWarnings.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+OSTRZEŻENIE
+ROZUMIEM
+Wykryto <b>zintegrowaną kartę graficzną Intel</b> która <b>nie jest wspierana</b>./n<b>Zalecane</B> jest użycie <b>niezintegrowanej</b> karty graficznej./nGra może <b>często wyrzucać</b> przy aktualnym sprzęcie.
+Nie można znaleźć ważnej aktualizacji systemu, którego wymaga czat głosowy. Proszę się upewnić, czy wersja twojego systemu operacyjnego Windows jest w pełni aktualna, w przeciwnym razie gra może niepoprawnie funkcjonować. Jeśli problem nadal występuje, proszę się skontaktować z naszą pomocą techniczną po dalszą pomoc.
```
Comment (SandeQ) [inline Translations/pl/MenuWarnings.txt]:
```suggestion
Nie można znaleźć ważnej aktualizacji systemu, której wymaga czat głosowy. Upewnij się, że masz zainstalowane najnowsze aktualizacje systemu Windows, w przeciwnym razie gra może nie funkcjonować poprawnie. Jeśli problem nadal występuje, prosimy o kontakt z naszą pomocą techniczną w celu uzyskania dalszej pomocy.
```

File: Translations/pl/MenuWarnings.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+OSTRZEŻENIE
+ROZUMIEM
+Wykryto <b>zintegrowaną kartę graficzną Intel</b> która <b>nie jest wspierana</b>./n<b>Zalecane</B> jest użycie <b>niezintegrowanej</b> karty graficznej./nGra może <b>często wyrzucać</b> przy aktualnym sprzęcie.
```
Comment (SandeQ) [inline Translations/pl/MenuWarnings.txt]:
```suggestion
Wykryto <b>zintegrowaną kartę graficzną firmy Intel</b> która <b>nie jest wspierana</b>./n<b>Zalecane</b> jest użycie <b>dedykowanej</b> karty graficznej./nZ aktualną kartą graficzną poprawne działanie gry <b><color=red>nie jest gwarantowane</color></b>.
```

File: Translations/pl/MainMenu.txt [hunk 3]
```diff
@@ -65,3 +65,12 @@ GŁOŚNOŚĆ OGÓLNA
 MUZYKA W MENU
 CZAT GŁOSOWY
 EFEKTY DŹWIĘKOWE
+USTAWIENIA ROZGRYWKI
+SZYBSZE WPROWADZENIE
+Tekst wprowadzający do klasy zniknie o wiele szybciej.
+SCP-079 - SPOSÓB ZMIANY WIDOKU
+Naciśnij raz, aby przełączyć tryb kamery.
```
Comment (SandeQ) [inline Translations/pl/MainMenu.txt]:
```suggestion
Naciśnij raz, aby przełączyć się pomiędzy trybami kamery.
```

File: Translations/pl/MainMenu.txt [hunk 4]
```diff
@@ -65,3 +65,12 @@ GŁOŚNOŚĆ OGÓLNA
 MUZYKA W MENU
 CZAT GŁOSOWY
 EFEKTY DŹWIĘKOWE
+USTAWIENIA ROZGRYWKI
+SZYBSZE WPROWADZENIE
```
Comment (SandeQ) [inline Translations/pl/MainMenu.txt]:
```suggestion
SZYBSZE OPISY
```

File: Translations/pl/MainMenu.txt [hunk 5]
```diff
@@ -65,3 +65,12 @@ GŁOŚNOŚĆ OGÓLNA
...
+Tekst wprowadzający do klasy zniknie o wiele szybciej.
```
Comment (SandeQ) [inline Translations/pl/MainMenu.txt]:
```suggestion
Opis aktualnej klasy przy rozpoczęciu rundy zniknie o wiele szybciej.
```

File: Translations/pl/MainMenu.txt [hunk 6]
```diff
@@ -65,3 +65,12 @@ GŁOŚNOŚĆ OGÓLNA
 MUZYKA W MENU
 CZAT GŁOSOWY
 EFEKTY DŹWIĘKOWE
+USTAWIENIA ROZGRYWKI
+SZYBSZE OPISY
+Opis aktualnej klasy przy rozpoczęciu rundy zniknie o wiele szybciej.
+SCP-079 - SPOSÓB ZMIANY WIDOKU
+Naciśnij raz, aby przełączyć się pomiędzy trybami ruchu kamery.
+SCP-079 - POST-PROCESSING
+Jeśli wyłączony, efekty na kamerze zostaną znacząco zmniejszone.
```
Comment (MichalPetryka) [inline Translations/pl/MainMenu.txt]: "Gdy wyłączony"?

File: Translations/pl/MenuWarnings.txt [hunk 7]
```diff
@@ -0,0 +1,4 @@
+OSTRZEŻENIE
+ROZUMIEM
+Wykryto <b>zintegrowaną kartę graficzną firmy Intel,</b> która <b>nie jest wspierana</b>./n<b>Zalecane</b> jest użycie <b>dedykowanej</b> karty graficznej./nZ aktualną kartą graficzną poprawne działanie gry <b><color=red>nie jest gwarantowane</color></b>.
+Nie można znaleźć ważnej aktualizacji systemu, której wymaga czat głosowy. Upewnij się, że masz zainstalowane najnowsze aktualizacje systemu Windows, w przeciwnym razie gra może nie funkcjonować poprawnie. Jeśli problem nadal występuje, prosimy o kontakt z naszą pomocą techniczną w celu uzyskania dalszej pomocy.
```
Comment (MichalPetryka) [inline Translations/pl/MenuWarnings.txt]: "dalszego wsparcia" to avoid repeating "pomocy"?

**Pull request merged**